src/Inkluzitron/Program.cs

```diff
@@ -87,6 +88,15 @@ static private async Task MainAsync(string[] args)
                 .AddSingleton<BdsmGraphPaintingStrategy>()
                 .AddSingleton<UsersService>()
                 .AddSingleton<KisSettings>()
+                .AddSingleton<ScheduledTasksService>()
```

Comment:
Když vidím kolik toho tu je, tak bych raději udělal extension metodu `AddVote(...)`, která by to popřidávala. tak jak to má po vzoru .NET/ASP.NET Core

Reply:
Vote věci jsem schoval do vlastní extension metody nad IServiceCollection.